### DISCUSSION TITLE
Kapsamlı Rework

### DISCUSSION THREAD

**Discussion by denizubu** [Ludeon/RimWorld-Turkish#408] [General]
Şu an kapsamlı bir rework üzerine çalışıyorum. 

Hem core oyunda hem de DLC çevirilerinde ciddi çeviri hataları/eksikleri var. Örneğin oyundaki sabit bir cismi tanımlayan bazı kelimelerin farklı farklı çevirileri var (muhtemelen farklı kişiler farklı zamanlarda çeviri yaptığı için). Bu da oyun içinde uyumsuzluğa sebep oluyor. Ayrıca bazı yerlerde oyundaki bağlamdan tamamen kopuk çeviriler yapılmış (muhtemelen google translate). Bu gibi çeviri hataları, yazım/noktalama ve dil bilgisi hataları gibi sıkıntıları düzeltmek için çalışıyorum.

Amacım şimdiye kadar olan bütün içeriği tam ve düzgün bir çeviri haline getirmek. Aslında bir nevi editörlük yapmak.

Gereken kısımlarda sizlerin de yardımına başvurmak üzere discord kanalımızda olacağım.

**Comment by denizubu** [comment]
Ana içerik reworkünü tamamladım. Royalty reworküne geçiyorum. Bütün rework komple tamamlanıp oynayarak test edilmeden buraya yüklemeyeceğim ama süreç ile ilgili haberdar edeceğim.

**Reply by SepuLturas1368** [reply]
tahmini ne zaman biter acaba?

**Reply by denizubu** [reply]
Kesin bi tarih veremiyorum ama işin büyük kısmı ana içerikti zaten. Yılbaşına kadar tamamlamayı umuyorum.

**Comment by denizubu** [comment]
Royalty tamam. Ideology reworküne geçiyorum.

**Reply by SepuLturas1368** [reply]
bitmesine ne kadar var acaba. Canım baya rimworld çekiyor ama cahil olduğum için ingilizce oynayamıyorum :3

**Reply by denizubu** [reply]
Maalesef hem iş hem üniversite sebebiyle çok sık ilgilenemiyorum. Bu sebeple kesin bi tarih veremiyorum. Yılbaşına kadar bitiririm diyordum ama daha sürecek gibi. Fakat benim repomdan ( https://github.com/denizubu/RimWorld-Turkish ) indirip oynayabilirsin. Tamamlanmış olmasa da şu anki çeviriden daha güncel durumda.

**Reply by SepuLturas1368** [reply]
tamamdır teşekkür ederim

**Reply by SepuLturas1368** [reply]
atmış olduğunuz linkteki çeviriyi data klasöründekilerle değiştirdim ama yazılarda herhangi bir değişiklik olmadı. sebebini biliyormusunuz acaba

**Reply by denizubu** [reply]
> atmış olduğunuz linkteki çeviriyi data klasöründekilerle değiştirdim ama yazılarda herhangi bir değişiklik olmadı. sebebini biliyormusunuz acaba

Tüm dosyalara aynı ismi verdiniz mi? Mesela indirdiğiniz Core dosyasının adını atıyorum "Çeviri" yaptıysanız diğerlerine de aynı adı vermeniz lazım. Ayrıca yeni oyuna başlamanız gerekebilir.

**Reply by SepuLturas1368** [reply]
yok direk zip dosyasının içindeki 5 dosyayı data klasörüne attım başka bir şey yapmadım.

**Reply by denizubu** [reply]
> yok direk zip dosyasının içindeki 5 dosyayı data klasörüne attım başka bir şey yapmadım.

Öyle çalışmaz hocam. İndirdiğiniz zipdeki Core dosyasının adını değiştirin. Atıyorum "Türkçe" yaptınız. Bunu Rimworld/Data/Core/Languages klasörüne yapıştırın. Aynı işlemi diğer dosyalar için de yapın. Hepsi Data içerisindeki ilgili klasörün Languages dosyasına atılacak ve hepsinin adı aynı olacak.

**Comment by denizubu** [comment]
Biotech DLC'sine geçiyorum

**Reply by ernxwsdm** [reply]
Eline sağlık.

**Comment by denizubu** [comment]
Biyotech tamam. Anomaliye geçiyorum.

**Comment by denizubu** [comment]
Daha önceden atladığım birkaç dosyaya bakıyorum ve son düzeltmeleri yapıyorum. Hedefim bu haftaiçi bunları bitirip haftasonu çeviriyi test etmek. En geç 2 hafta içerisinde bütün iş bitmiş olacak. Teste geçerken buraya yazacağım.

**Comment by denizubu** [comment]
Teste geçiyorum.

**Comment by denizubu** [comment]
Test tamam. Repoyu güncelliyorum. Lütfen hataları bildirin. İyi oyunlar.
